lambda/getQueryString.ts

```diff
@@ -0,0 +1,21 @@
+export const getQueryString = ({
+	binIntervalMinutes,
```

Comment:
**problem:** use the type HistoricalDataTimeSpan here so you ensure that these property names align across the codebase (and it's clear where they are coming from)

Reply:
Can I make the 'expiredMinutes' property optional to use this type? I'm not using it anywhere now.

Reply:
Yes, you can use for example `Pick<HistoricalDataTimeSpan, 'binIntervalMinutes' | 'durationHours'>` to construct a new type based on `HistoricalDataTimeSpan` that only has the two required properties.

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys